### PR TITLE
libredblack: Pass CFLAGS

### PR DESCRIPTION
**Pull request by neheb**
CFLAGS were not being passed. This was breaking builds with ASLR.

Signed-off-by: Rosen Penev <rosenp@gmail.com>

Maintainer: @mislavn
Compile tested: mvebu

### CODE REVIEW

File: libs/libredblack/Makefile [hunk 1]
```diff
@@ -37,6 +39,7 @@ define Package/libredblack/description
 endef
 
 CONFIGURE_ARGS += --without-rbgen
+CONFIGURE_VARS += lt_cv_prog_cc_pic=$(FPIC)
```
Comment (karlp) [inline libs/libredblack/Makefile]: not mentioned in the commit, you're doing more here again.

File: libs/libredblack/Makefile [hunk 2]
```diff
@@ -19,8 +19,10 @@ PKG_MAINTAINER:=Mislav Novakovic <mislav.novakovic@sartura.hr>
 PKG_LICENSE:=GPL-2.0-or-later
 PKG_LICENSE_FILES:=COPYING
 
-PKG_BUILD_PARALLEL:=1
+PKG_FIXUP:=autoreconf
 PKG_INSTALL:=1
+PKG_BUILD_PARALLEL:=1
+PKG_REMOVE_FILES:=autogen.sh aclocal.m4
```
Comment (karlp) [inline libs/libredblack/Makefile]: not mentioned in the commit, you're doing extra work in this commit again.

**Comment by neheb** [comment]
Patch is not needed. Turns out the missing link was autoreconf.

**Comment by BKPepe** [comment]
Can we cherry-pick it to branch `openwrt-19.07`?

**Pull request merged**